src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/getUserById';
```

Comment:
[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has `null` or `undefined` value. You should prepare data in one place (App component) and pass it to child components. Instead of calling `getUserById` inside the `TodoInfo` component, consider moving this logic to the parent component and passing the user object as a prop to `TodoInfo`.